dump/README.md

```diff
@@ -0,0 +1,72 @@
+# dump
+
+Package dump can dump a Go data structure to Go source file, so that it can be statically embeded into other code.
+
+# Project Status
+* UnsafePointer is not supported yet
+* cyclic pointer is not supported yet
+
+# Implementation Notes
+
+## Aliase
+
+Primitive pointer and reused pointer will be aliased.
```

Comment:
"Aliases to primitives and structs will be preserved in the objects described by the generated code."